src/overflow/explainer.md

```diff
@@ -1,6 +1,7 @@
 ---
 created: 2023-12-14
 title: CSS Overflow - Broad Research
+progress: 👍🏼
```

Comment:
I think the "👍" is preferred for screen readers as it just reads "bullet thumbs up" as opposed to "bullet thumbs up medium light skin tone"